src/Entity/Course.php

```diff
@@ -123,7 +124,7 @@ public function __construct(
         $this->content = $content;
         $this->ratings = new ArrayCollection();
         $this->tags = new ArrayCollection();
-        $this->summary = new ArrayCollection();
+        $this->summary = $summary;
```

Comment:
jj pravda, summary je OneToOne, takže to asi nebude ArrayCollection :)

Reply:
opraveno